src/LndClient/Data/GetInfo.hs

```diff
@@ -20,3 +25,14 @@ instance FromGrpc GetInfoResponse GRPC.GetInfoResponse where
       <$> fromGrpc (GRPC.getInfoResponseIdentityPubkey x)
       <*> fromGrpc (GRPC.getInfoResponseSyncedToChain x)
       <*> fromGrpc (GRPC.getInfoResponseSyncedToGraph x)
+
+instance C2.FromGrpc GetInfoResponse LnGRPC.GetInfoResponse where
+  fromGrpc x =
+    GetInfoResponse
+      <$> mPubKey
+      <*> mSyncedToChain
+      <*> mSyncedToGraph
+    where
+      mSyncedToChain = maybeToRight (FromGrpcError "SyncedToChain is not set") $ x ^? LnGRPC.syncedToChain
```

Comment:
For scalar fields `^?` mightbe redundant, I think in most cases `^.` can be used + `maybe'*` lens for the messages which actually can be unset. But please verify it youself, it's just my assumptions

Reply:
replaced it with ^.
for maybe fields maybe' lens can be used, so done